met-api/src/met_api/resources/language.py

```diff
@@ -101,42 +121,15 @@ def post(language_id, tenant_id):
                 str(detail), HTTPStatus.INTERNAL_SERVER_ERROR
             ) from e
 
-
-@cors_preflight('DELETE, OPTIONS')
-@API.route('/mappings/<int:language_mapping_id>')
-class DeleteLanguageMapping(Resource):
-    """Resourse for removing language-tenant relationships."""
-
     @staticmethod
-    # @_jwt.requires_auth
-    # @cross_origin(origins=allowedorigins())
-    def delete(language_mapping_id):
+    @_jwt.requires_auth
+    @cross_origin(origins=allowedorigins())
+    def delete(language_id, tenant_short_name):
         """Remove a language mapping from a tenant."""
         try:
-            success = LanguageService.remove_language_mapping_from_tenant(language_mapping_id)
-            if success:
-                return 'Successfully deleted language-tenant mapping', HTTPStatus.NO_CONTENT
-            raise ValueError('Language-tenant mapping not found')
+            LanguageService.remove_language_mapping_from_tenant(language_id, tenant_short_name)
+            return {'status': 'success', 'message': 'Tenant language deleted successfully'}, HTTPStatus.OK
```

Comment:
You could also return [HTTP 204 No Content](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204) to indicate that the resource no longer exists, unless the `message` is planned to be utilized by the app

Reply:
I considered this for sure! Do you think there's a downside to having a message & 200 code returned on delete? I don't necessarily think that a 204 on delete is super idiomatic across the web